README.md

```diff
@@ -76,6 +85,13 @@ and the following Typescript types:
 - `GoResult<T> = { data: T; success: true }`
 - `GoResultSuccess<E extends Error = Error> = { error: E; success: false }`
 - `GoResultError<T, E extends Error = Error> = GoResultSuccess<T> | GoResultError<E>`
+- `PromiseOptions { readonly retries?: number; readonly retryDelayMs?: number; readonly timeoutMs?: number; }`
+
+The default values for `PromiseOptions` are:
+
+```ts
+{ retries: 0, retryDelay: 200, timeoutMs: 0 }
```

Comment:
Maybe we can add that timeout 0 means no timeout. Even though I guess it's quite obvious.